.deploy.yml

```diff
@@ -28,7 +28,12 @@ jobs:
   condition: eq(variables['Build.Reason'], 'PullRequest')
   steps:
     - powershell: ./build.ps1
-
+    env:
+      OP_BUILD_USER_TOKEN: $(OpBuildUserToken)
```

Comment:
we don't need `OP_BUILD_USER_TOKEN` for build, it is to support running config migration script in impact testing